pages/common/git-diff.md

```diff
@@ -25,3 +25,7 @@
 - Create a patch file:
 
 `git diff > {{target_file}}.patch`
+
+- Compare a single file between two branches / commits:
+
+`git diff {{branch_1}}..{{branch_2}} [--] {{/path/to/file}}`
```

Comment:
Are the `--` optional? I thought they were required.

Also, could you remove the preceding slash in `/path/to/file` making it `path/to/file`, please?

Thanks 😺